activejob/test/jobs/multiple_kwargs_job.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require_relative "../support/job_buffer"
+
+class MultipleKwargsJob < ActiveJob::Base
+  def perform(argument1:, argument2:)
+    JobBuffer.add("Job with argument1: #{argument1}, argument2: #{argument2}")
```

Comment:
I don't think this line is required but I added it as it was added in other classes. I can remove it if need be.